intranet/apps/announcements/forms.py

```diff
@@ -37,6 +34,36 @@ class Meta:
         fields = ["title", "author", "content", "groups", "expiration_date", "notify_post", "notify_email_all", "update_added_date", "pinned"]
 
 
+class AnnouncementEditForm(forms.ModelForm):
+    """A form for generating an announcement."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["expiration_date"].help_text = "By default, announcements expire after two weeks. To change this, click in the box above."
+
+        self.fields["notify_post_resend"].help_text = "If this box is checked, students who have signed up for notifications will receive an email."
+
+        self.fields["notify_email_all_resend"].help_text = (
+            "This will send an email notification to all of the users who can see this post. This option "
```

Comment:
```suggestion
            "This will resend an email notification to all of the users who can see this post. This option "
```

Maybe make it clearer by adding "resend"?